### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by felixZzzxy** [EDAPINENUT/ppflow#8]
Hi, I have download the ppbench2024 and I found some PDB dir have peptide.pdb and peptide_fix.pdb, the same for receptor. What's the difference between these two files ? Thanks for your attention.

**Comment by EDAPINENUT** [comment]
Yes, we use foldx to refine the structure. However, we find it still not well-refined. 
We have discussed the problem with the authors at https://github.com/THUNLP-MT/PepGLAD.
The structures are well-selected, and we are processing the data as the input for the model.
If the re-processed ones are released, we will inform you A.S.A.P.
